packages/calcite-components/src/components/rating/rating.tsx

```diff
@@ -259,31 +239,26 @@ export class Rating
           <fieldset class="fieldset" disabled={this.disabled}>
             <legend class="visually-hidden">{this.messages.rating}</legend>
             {this.starsMap.map(
-              ({
-                average,
-                checked,
-                focused,
-                fraction,
-                hovered,
-                id,
-                idx,
-                partial,
-                selected,
-                value,
-              }) => {
+              ({ average, checked, fraction, hovered, id, partial, selected, value, tabIndex }) => {
                 return (
                   <label
                     class={{
                       star: true,
-                      focused,
                       selected,
                       hovered,
                       average,
                       partial,
                     }}
+                    data-value={value}
                     htmlFor={id}
+                    onClick={this.handleLabelClick}
+                    onFocus={this.handleLabelFocus}
+                    onKeyDown={this.handleLabelKeyDown}
                     onPointerDown={this.handleLabelPointerDown}
                     onPointerOver={this.handleLabelPointerOver}
+                    tabIndex={tabIndex}
```

Comment:
Does making `label` have any impact on how screen readers interpret the value?

Reply:
the screen reader experience has no impact in VoiceOver.  cc @geospatialem 

I do see a lot of intermediate states with VoiceOver where visually hidden area are being focused when user is navigating using voiceover commands + rightArrow.  we need to add `aria-hidden` attribute to remove the visually hidden elements from accessibility tree. This can be done in a different PR though, wanted to confirm similar behavior with NVDA/JAWS before diving in.

Reply:
One addition could be announcing to AT user when user reset the selection ( not  available in native behavior).

Reply:
Not following the logic above, is this to remove the `label`?